.github/workflows/appsignals-e2e-ec2-test.yml

```diff
@@ -34,6 +33,15 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set Parameter APP_SIGNALS_CW_AGENT_RPM
```

Comment:
nit: Make this more obvious what it is doing, for example "Set CW Agent RPM environment variable"

Reply:
Changing based on that